sdk/examples/how_tos/native_tokens/mint.rs

```diff
@@ -51,7 +49,11 @@ async fn main() -> Result<()> {
         let block_id = account
             .retry_transaction_until_included(&transaction.transaction_id, None, None)
             .await?;
-        println!("Block included: {}/block/{}", var("EXPLORER_URL").unwrap(), block_id);
+        println!(
+            "Transaction included: {}/block/{}",
```

Comment:
These changes in the native_tokens how tos are now different than in https://github.com/iotaledger/iota-sdk/tree/develop/documentation/sdk/docs/how_tos/native_tokens, so either undo or change also in the .mdx files and binding examples

Reply:
right :+1:

Reply:
the phrase "block included" isn't really good

Reply:
What's different between block and transaction there?

Reply:
for me: blocks (tangle layer, can be referenced, or not), transactions (ledger layer, can be included, or not)

Reply:
> we retry until the transaction is included.

The function is called like that because a transaction can be in multiple blocks and it doesn't matter which of these get referenced by a milestone (which for me also means that the block is included in a milestone :trollface:)

But I'm fine if you want to change it, just change it also in the how tos .mdx and bindings examples